wagtail/contrib/redirects/migrations/0010_populate_hash.py

```diff
@@ -13,7 +12,7 @@ def migrate_forwards(apps, schema_editor):
 
     for redirect in Redirect.objects.all().only("id", "old_path", "site_id").iterator():
 
-        redirect.hash = get_redirect_hash(redirect.old_path)
+        redirect.hash = Redirect.get_redirect_hash(redirect.old_path)
```

Comment:
@fbarnea I don't think this will work here, as you only have a 'built from migration data' version of the `Redirect` model. You'd need to import the real model class here separately in order to access this method.

Reply:
Of course. I've pushed a fix for this.

Reply:
@fbarnea Do we need the imported version as a separate thing?

The iterator is now using the imported version, rather than what the migration system has for the table at this point in time for migrations.